code/ATMOSPHERICS/atmospherics.dm

```diff
@@ -289,7 +289,9 @@ Pipelines + Other Objects -> Pipe network
 	return FALSE
 
 /obj/machinery/atmospherics/cultify()
-	if(src.invisibility != INVISIBILITY_MAXIMUM)
+	if(density)
+		..()
+	else if(src.invisibility != INVISIBILITY_MAXIMUM)
```

Comment:
:csgrad:

Reply:
Should I just lose the ``if()`` or what?

Reply:
Yeah